tests/test_cluster.py

```diff
@@ -46,39 +47,38 @@ def test_install():
     )
 
 
-def test_basic_run():
+@pytest.mark.clustertest
+def test_basic_run(cpu):
     # Create temp file where fn's will be stored
     test_cmd = "echo hi"
-    hw = cluster(name="^rh-cpu")
-    hw.up_if_not()
-    res = hw.run(commands=[test_cmd])
+    cpu.up_if_not()
+    res = cpu.run(commands=[test_cmd])
     assert "hi" in res[0][1]
 
 
-def test_restart_grpc():
-    hw = cluster(name="^rh-cpu")
-    hw.up_if_not()
-    codes = hw.restart_grpc_server(resync_rh=False)
+@pytest.mark.clustertest
+def test_restart_grpc(cpu):
+    cpu.up_if_not()
+    codes = cpu.restart_grpc_server(resync_rh=False)
     assert codes
 
 
-def test_on_same_cluster():
-    hw = cluster(name="^rh-cpu").up_if_not()
-    hw.restart_grpc_server()
-    hw.up_if_not()
-
-    hw_copy = cluster(name="^rh-cpu")
+@pytest.mark.clustertest
+def test_on_same_cluster(cpu):
+    hw_copy = cpu.copy()
+    cpu.restart_grpc_server()
+    cpu.up_if_not()
 
-    func_hw = rh.function(is_on_cluster).to(hw)
-    assert func_hw(hw)
+    func_hw = rh.function(is_on_cluster).to(cpu)
+    assert func_hw(cpu)
     assert func_hw(hw_copy)
 
 
-def test_on_diff_cluster():
-    hw = cluster(name="^rh-cpu").up_if_not()
-    diff_hw = rh.cluster(name="test-byo-cluster").up_if_not()
+@pytest.mark.clustertest
+def test_on_diff_cluster(cpu, a10g):
```

Comment:
We don't need an a10 for this, could we add a fixture for another cpu cluster (e.g. the one we use in the byo tests with a different boot image)? We often need to use two in tests anyway.